commons/src/main/java/io/ballerina/stdlib/graphql/commons/utils/SdlSchemaStringGenerator.java

```diff
@@ -331,4 +375,12 @@ private static Boolean isDefaultDirective(Directive directive) {
     private static String getFormattedString(String format, String... args) {
         return String.format(format, (Object[]) args);
     }
+
+    private static String removeLeadingNewLines(String description) {
+        int i = 0;
+        while (i < description.length() && String.valueOf(description.charAt(i)).equals(LINE_SEPARATOR)) {
```

Comment:
When LINE_SEPARATOR takes the value "\r\n" will this work ? because we are using `charAt(i)` here which check only on character at a time

Reply:
Removed this function